spec/units.yaml

```diff
@@ -1,238 +1,272 @@
     domain: energy consumption
   g:
-    label: gram
     description: Mass measured in grams
     domain: mass
   kg:
-    label: kilogram
     description: Mass measured in kilograms
     domain: mass
   lbs:
-    label: pound
     description: Mass measured in pounds
     domain: mass
   V:
-    label: volt
     description: Electric potential measured in volts
     domain: electric potential
   A:
-    label: ampere
     description: Electric current measured in amperes
     domain: electric current
   Ah:
-    label: ampere hours
     description: Electric charge measured in ampere hours
     domain: electric charge
   ms:
-    label: millisecond
     description: Time measured in milliseconds
     domain: time
+    denominator: 1000
+    allowed-datatypes: ['numeric']
   s:
-    label: second
     description: Time measured in seconds
     domain: time
+    allowed-datatypes: ['numeric']
   min:
-    label: minute
     description: Time measured in minutes
     domain: time
+    multiplier: 60
+    allowed-datatypes: ['numeric']
   h:
-    label: hour
     description: Time measured in hours
     domain: time
+    multiplier: 3600
+    allowed-datatypes: ['numeric']
   day:
-    label: days
     description: Time measured in days
     domain: time
+    multiplier: 86400
+    allowed-datatypes: ['numeric']
   weeks:
-    label: weeks
     description: Time measured in weeks
     domain: time
+    multiplier: 604800
+    allowed-datatypes: ['numeric']
   months:
-    label: months
     description: Time measured in months
     domain: time
+    # Multiplier based on 30 day month
+    multiplier: 2592000
+    allowed-datatypes: ['numeric']
   years:
-    label: years
     description: Time measured in years
     domain: time
+    # Multiplier based on 365 day year
+    multiplier: 31536000
+    allowed-datatypes: ['numeric']
+  iso8061:
+    description: Point in time expressed as string according to ISO 8601
+    domain: point in time
+    allowed-datatypes: ['string']
+    # Special conversion method 1so8601 -> timestamp needed
   UNIX Timestamp:
-    label: UNIX Timestamp
     description: Unix time is a system for describing a point in time. It is the number of seconds that have elapsed since the Unix epoch, excluding leap seconds.
     domain: point in time
+    # Conversion custom , i.e this file does not specify how conversion to iso8061 happens
+    conversion: custom
+    allowed-datatypes: ['uint32','uint64','int64']
   mbar:
-    label: millibar
     description: Pressure measured in millibars
     domain: pressure
+    multiplier: 100
   Pa:
-    label: pascal
     description: Pressure measured in pascal
     domain: pressure
   kPa:
-    label: kilopascal
     description: Pressure measured in kilopascal
     domain: pressure
+    multiplier: 1000
+  psi:
+    description: Pressure measured in pounds per square inch
+    domain: pressure
+    multiplier: 6894.7572931783
```

Comment:
Would be 6894.757 if we would follow the recommendation to use 7 significant digits in the document referenced by Adnan